rootfs/usr/share/inputplumber/devices/60-xbox_360_gamepad.yaml

```diff
@@ -0,0 +1,272 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/ShadowBlip/InputPlumber/main/rootfs/usr/share/inputplumber/schema/composite_device_v1.json
+# Schema version number
+version: 1
+
+# The type of configuration schema
+kind: CompositeDevice
+
+# Name of the composite device mapping
+name: Xbox 360 Gamepad
+
+# Only use this profile if *any* of the given matches matches. If this list is
+# empty then the source devices will *always* be checked.
+# /sys/class/dmi/id/product_name
+matches: []
+
+# One or more source devices to combine into a single virtual device. The events
+# from these devices will be watched and translated according to the key map.
+source_devices:
+  - group: gamepad
+    evdev:
+      name: "Generic X-Box pad"
+
+  - group: gamepad
+    evdev:
+      vendor_id: "0x0079"
+      product_id: "0x18d4"
+
+  - group: gamepad
+    evdev:
+      vendor_id: "0x03eb"
+      product_id: { "0xff01", "0xff02" }
```

Comment:
This does not conform to the YAML schema. If you were trying to glob match on the id, it would be like:

```yaml
product_id: "{0xff01,0xff02}"
```

From the Legion Go profile, it also looks like there's no `0x` prefix the way we're parsing the product id.

Reply:
-_-